script/misc-helper.ts

```diff
@@ -332,31 +332,52 @@ async function getCIHelper(): Promise<CIHelper> {
         const glue = new GitHubGlue();
         await glue.addPRComment(pullRequestURL, comment);
     } else if (command === "set-app-token") {
-        if (commander.args.length !== 1) {
+        if (commander.args.length < 1) {
```

Comment:
Is this check still needed?  The code will run with no args won't it (assuming slice will have no issues)?  If it stays, maybe the message should be `unexpected` to `missing`.

Reply:
Well spotted. Will fix.